docs/how-to/build/inferencing.md

```diff
@@ -101,7 +101,7 @@ GCC 4.x and below are not supported.
 | OS/Compiler | Supports VC  | Supports GCC     |  Supports Clang  |
 |-------------|:------------:|:----------------:|:----------------:|
 |Windows 10   | YES          | Not tested       | Not tested       |
-|Linux        | NO           | YES(gcc>=4.8)    | Not tested       |
+|Linux        | NO           | YES(gcc>=8)    | Not tested       |
```

Comment:
Our pipelines only tested ORT code with gcc 8 and 9. I know 10 should also work.